03_launch_mgmt_cluster.sh

```diff
@@ -176,6 +194,7 @@ sudo su -l -c 'minikube ssh sudo ip addr add 172.22.0.2/24 dev eth2' "${USER}"
 launch_baremetal_operator
 apply_bm_hosts
 launch_cluster_api_provider_baremetal
```

Comment:
This should be last, the controller does not start properly otherwise, since the capi CRDs do not exist